### ISSUE TITLE
Can't download systems

### ISSUE THREAD

**Issue by kerunaru** [pokebyte/CocoaMSX#32]
Every time I try to refresh the list I get the following error…

<img width="710" alt="captura de pantalla 2015-10-27 a las 15 23 17" src="https://cloud.githubusercontent.com/assets/94023/10760813/bba34d72-7cbe-11e5-9544-b4648f7b08a3.png">

I also get error when try to check for updates…

<img width="532" alt="captura de pantalla 2015-10-27 a las 15 25 13" src="https://cloud.githubusercontent.com/assets/94023/10760888/04763316-7cbf-11e5-8f16-76a7bbeea88f.png">

I've found a suspicious log on console.app

```
27/10/15 15:26:49,312 CocoaMSX[59142]: App Transport Security has blocked a cleartext HTTP (http://) resource load since it is insecure. Temporary exceptions can be configured via your app's Info.plist file.
```

This is logged every time CocoaMSX tries to connect to the Internet.

I'm on a 15" Retina Display MacBook Pro on OS X 10.11.
Core i7 · 8GB RAM

**Comment by pokebyte** [comment]
Thanks for reporting. This was fixed in 021d060917d263e298c5e266b47106af06c38835, and has to do with the new security features in El Capitan.

**Comment by kerunaru** [comment]
:+1:
